.github/workflows/release-binaries.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  push:
+    tags:
+      # Push events to matching vX.X.X, i.e. v0.1.0
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+name: Create Binaries Release
+
+# Note: According to https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables,
+# the value of {{ github.ref }} is "refs/tags/<tag>", so we have to remove that prefix everywhere.
+jobs:
+  build-and-release:
+    name: Build and Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.14'
+    - run: go version
+
+    - name: Build Go binaries
+      shell: bash
+      run: ./build/build-binaries.sh -v "$(echo '${{ github.ref }}' | sed -e 's|^refs/tags/||')"
+
+    # From https://github.com/github/hub#github-actions
+    - name: Create GitHub Release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
whose token is this?

Reply:
See https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token:
"GitHub automatically creates a GITHUB_TOKEN secret to use in your workflow. You can use the GITHUB_TOKEN to authenticate in a workflow run.

Before each job begins, GitHub fetches an installation access token for the job. The token expires when the job is finished."